Mathlib/RingTheory/MvPowerSeries/PiTopology.lean

```diff
@@ -235,6 +242,46 @@ theorem instIsUniformAddGroup [AddGroup R] [IsUniformAddGroup R] :
 
 end Uniformity
 
+section ext
+
+variable {σ τ R S : Type*} [TopologicalSpace R] [TopologicalSpace S]
+
+lemma continuous_linearMap_ext [CommSemiring R] [AddCommMonoid S] [Module R S] [T2Space R]
+    [T2Space S] {f g : MvPowerSeries σ R →ₗ[R] S}
+    (hf : Continuous f) (hg : Continuous g) (h : ∀ i, f (monomial R i 1) = g (monomial R i 1))
+    (P : MvPowerSeries σ R) : f P = g P := by
+  rw [as_tsum P, Summable.map_tsum ⟨_, hasSum_of_monomials_self _⟩ _ hf,
```

Comment:
You could pass through  the density of polynomials.

Reply:
That one wants `CommRing` everywhere (it can surely be generalized it requires to generalize `HasEval` or something like that).